src/cr/cube/matrix.py

```diff
@@ -55,22 +85,70 @@ def rows(self):
 
     @lazyproperty
     def table_base(self):
+        """np.int64 scalar or a 1D or 2D ndarray of np.int64 representing table base.
+
+        A multiple-response (MR) dimension produces an array of table-base values
+        because each element (subvariable) of the dimension represents a logically
+        distinct question which may not have been asked of all respondents. When both
+        dimensions are MR, the return value is a 2D ndarray and there is a distinct
+        table-base value for each "cell" of the matrix. A CAT_X_CAT matrix produces
+        a scalar value for this property.
+        """
         return self.table_base_unpruned[
             np.ix_(self._visible_rows_mask, self._visible_cols_mask)
         ]
 
     @lazyproperty
     def table_base_unpruned(self):
+        """np.int64 scalar or a 1D or 2D ndarray of np.int64 representing table base.
+
+        This version includes hidden vectors, those with either a hide transform on
+        their element or that have been pruned (because their base (N) is zero). This
+        does not affect a scalar value but when the return value is an ndarray, the
+        shape may be different than the array returned by `.table_base`.
+
+        A multiple-response (MR) dimension produces an array of table-base values
+        because each element (subvariable) of the dimension represents a logically
+        distinct question which may not have been asked of all respondents. When both
+        dimensions are MR, the return value is a 2D ndarray. A CAT_X_CAT matrix produces
+        a scalar value for this property.
+        """
         return self._unordered_matrix.table_base
 
     @lazyproperty
     def table_margin(self):
+        """np.float/int64 scalar or a 1D or 2D np.float/int64 ndarray table margin.
+
+        The table margin is the overall sample size of the matrix. This is the weighted
+        count of respondents who were asked both questions and provided a valid response
```

Comment:
I don't get what "both" means in this context? Which both questions? Did you mean "who were both asked the questions, and provided a valid response?". Or did I miss something altogether?